### PR TITLE
Jetpack compose engineering resources

### PR DESCRIPTION
**Pull request by wakoliVotes**
## Hacktoberfest
- Submitting as part of hacktoberfest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ List of resources and project samples for Engineering Students
   - [MySQL <a name="mysql"></a>](#mysql-)
   - [Java Programming <a name="java-programming"></a>](#java-programming-)
   - [Kotlin Programming <a name="kotlin-programming"></a>](#kotlin-programming-)
+  - [Jetpack Componse <a name="jetpack-compose"></a>](#jetpack-compose)
```
Comment (aaryan2134) [inline README.md:36]: Please correct the spelling

Comment (aaryan2134) [reply README.md:36]: Also, the link is not pointing to the section as well

**Review by aaryan2134 — CHANGES_REQUESTED** [review]
Please make these changes. Looks good otherwise
Thank you!

File: README.md (hunk 1 above)
Comment (wakoliVotes) [reply README.md]: Hello, completed
### Fix
- Typo in README
- Links to source
-
Thank You.

**Comment by aaryan2134** [comment]
Awesome! Great Work

**Pull request merged**